### ISSUE TITLE
Using Eager works, using local files does not

### ISSUE THREAD

**Issue by 2braincells2go** [HubSpot/offline#151]
Realize that this has been reported few times here, but no replies to those issues.

When I install using Eager, can get alert box and the chrome theme working. Tab shows.

When installing as on site using the manual install nothing works. There are no errors in developer console to indicate any issues.

Really like to use this in some school related projects but have to run files locally, cannot call files from Eager (firewall blocks those type of requests).

It appears the creator does not support issues here. Is there anyone who has this working with manual method? Maybe an example of using the manual method?

Thanks!

**Comment by zackbloom** [comment]
These are the files Eager includes: https://github.com/HubSpot/offline/blob/master/install.json#L5-L26

Perhaps you're missing a theme or language file?

**Comment by 2braincells2go** [comment]
Thanks for tip @zackbloom!

Will have to look at that. Just assumed that following manual instructions to place in a page would work without further coding.

**Comment by 2braincells2go** [comment]
I finally got a chance to check into this deeper.

Have followed the instructions here (where I first learned of offline.js):

https://www.gavick.com/blog/detect-offline-browser

So I am calling the offline.js and two css files in head tag along with the function to check connection every 5 seconds.

Does not work for me in Chore or IE.

Can use the Eager provided script in header and does work. So lost at what is wrong.

Thanks again for input.
